GameOfLife/GameOfLife/Services/FileIO.cs

```diff
@@ -13,36 +13,13 @@ public class FileIO : IFileIO
         private IRender _render;
         private IInputController _inputController;
         private IEngine _engine;
-        private int _numberOfFiles;
-        private string _filePath = @"C:\GameOfLife_SavedGames\";
         private string[] _stringField;
         private List<string> _stringList = new List<string>();
-        private bool _fileReadingError = false;
-        private bool _fileLoaded = false;
-        private bool _noSavedGames = false;
-        public bool FileReadingError
-        {
-            get => _fileReadingError;
-            set => _fileReadingError = value;
-        }
-        public string FilePath
-        {
-            get => _filePath;
-        }
-        public bool FileLoaded
-        {
-            get => _fileLoaded;
-            set => _fileLoaded = value;
-        }
-        public bool NoSavedGames
-        {
-            get => _noSavedGames;
-            set => _noSavedGames = value;
-        }
-        public int NumberOfFiles
-        {
-            get => _numberOfFiles;
-        }
+        public string FilePath { get; } = @"C:\GameOfLife_SavedGames\";
```

Comment:
Magic string => ' @"C:\GameOfLife_SavedGames\";'
Should be taken out of code. You can create static class for parameters and place data there.
One implementation shortcut/improvement is to put data in subfolder of folder where running application is located.
You can get that by:
AppDomain.CurrentDomain.BaseDirectory

Reply:
Made the location of the SavedGames folder relative to the project folder.